src/hooks/core/chat/useChannelGetInitialMessagesHook.ts

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 
+import {Platform} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Platform` from 'react-native' is a good addition to allow for platform-specific logic.
```diff
+ import {Platform} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->